client/app/visualizations/chart/index.js

```diff
@@ -1,10 +1,32 @@
 import {
-  some, extend, has, partial, intersection, without, contains, isUndefined,
+  some, extend, defaults, has, partial, intersection, without, contains, isUndefined,
   sortBy, each, pluck, keys, difference,
 } from 'underscore';
 import template from './chart.html';
 import editorTemplate from './chart-editor.html';
 
+const DEFAULT_OPTIONS = {
+  globalSeriesType: 'column',
+  sortX: true,
+  legend: { enabled: true },
+  yAxis: [{ type: 'linear' }, { type: 'linear', opposite: true }],
+  xAxis: { type: 'datetime', labels: { enabled: true } },
+  error_y: { type: 'data', visible: true },
+  series: { stacking: null, error_y: { type: 'data', visible: true } },
+  seriesOptions: {},
+  columnMapping: {},
+
+  numberFormat: '0,0[.]00000',
+  percentFormat: '0[.]00%',
+  dateTimeFormat: 'DD/MM/YYYY HH:mm',
```

Comment:
The default here should be based on the value in `clientConfig`.

Reply:
Ping.

Reply:
Sorry... Done